terraform/cyhy_reporter_ec2.tf

```diff
@@ -121,6 +122,7 @@ module "cyhy_reporter_ansible_provisioner" {
     "production_workspace=${local.production_workspace}",
     "ses_aws_region=${var.ses_aws_region}",
     "docker_compose_override_file_for_mailer=${var.reporter_mailer_override_filename}",
+    "ses_send_email_role=arn:aws:iam::${var.ses_account_id}:role/SesSendEmail-cyber.dhs.gov",
```

Comment:
Same question here as above.

Reply:
Fixed with a dramatic force push to remove the role name from the history.  See commit fdefaf7.

Reply:
👍